server/models/userCourseRegister.model.js

```diff
@@ -8,11 +8,11 @@ const userCourseRegister = new Schema({
         unique: true,
         required: true
     },
-    name: {
+    candidateName: {
         type: String,
         required: true
     },
-    userId: {
+    email: {
         type: String,
         required: true
     },
```

Comment:
`userId` missing. 
user with same `emailId` can apply for multiple courses with different `usernames`, so we need `userId` for proper identification.